src/main/java/kz/andersen/java_intensive_13/collector/CollectorManager.java

```diff
@@ -0,0 +1,41 @@
+package kz.andersen.java_intensive_13.collector;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.core.exc.StreamWriteException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CollectorManager<T> implements StateSerializer<T>{
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
if its static, so should be UPPER_CASE
but, I think it could be just non static)) please, check